regolith/helpers/u_contacthelper.py

```diff
@@ -16,9 +16,12 @@
 
 def subparser(subpi):
     subpi.add_argument("name", help="first name space last name in quotes")
-    subpi.add_argument("institution", help="person's institution.  Can be inf"
-                                            "short form such as columbiau and will "
-                                            "be retrieved from institutions collection")
+    subpi.add_argument("--id", help="id of the person, e.g., first letter first name "
+                                    "plus last name, but unique")
+    subpi.add_argument("--inst", help="person's institution.  Can be inf "
```

Comment:
maybe have a single letter abbreviation, ``-i`` for institution and ``-d` for id?

Also inf is a typo?

Reply:
done!